test/Microsoft.AspNetCore.Rewrite.Tests/MiddlewareTests.cs

```diff
@@ -16,7 +16,7 @@ public class MiddlewareTests
         [Fact]
         public async Task CheckRewritePath()
         {
-            var options = new RewriteOptions().AddRewrite("(.*)", "http://example.com/$1", skipRemainingRules: false);
+            var options = new RewriteOptions().AddRewrite("(.*)", "http://example.com$1", skipRemainingRules: false);
```

Comment:
Does this is affect parsed mod_rewrite/URL Rewrite rules? e.g. `RewriteRule (.*) http://example.com/$1`